pkg/contexts/ocm/accessmethods/wget/method.go

```diff
@@ -0,0 +1,151 @@
+package wget
+
+import (
+	"fmt"
+	"io"
+	"sync"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess"
+	"github.com/open-component-model/ocm/pkg/blobaccess/wget"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials/builtin/wget/identity"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi/accspeccpi"
+	"github.com/open-component-model/ocm/pkg/mime"
+	"github.com/open-component-model/ocm/pkg/runtime"
+)
+
+// Type is the access type for a blob on an http server .
```

Comment:
```suggestion
// Type is the access type for a blob on a http server.
```

Reply:
Well, this is really nitpicky, but since the 'h' of 'http' is commonly pronounced "age" and "age" starts with a vowel, I believe a**n** can actually be argued to be the correct article (of course, if you read it as 'hypertext transfer protocol', you are right).